### DISCUSSION TITLE
Local Index restricted to certain tag

### DISCUSSION THREAD

**Discussion by thoresson** [SkepticMystic/breadcrumbs#420] [Q&A]
I base my Zettelkasten on an hierarchy of topic notes, to which I link my atomic notes as children. 

To decide where a atomic note beat fits, I would like to have an outline of my topic hierarchy.

Is it possible to restrict the Local Index to only include notes with a certain tag? All topic notes are tagged _meta/topic_, and could be used as an selector for what to include and what to exclude from the index. 

/Anders

**Comment by michaelpporter** [comment]
This already works today — no extra setup needed. A `breadcrumbs` codeblock takes a `dataview-from` filter, so you can restrict the local index to a tag:

````
```breadcrumbs
type: tree
dataview-from: "#your-tag"
```
````

The traversal starts from the current note and only keeps descendants matching the filter. The parser is built-in (no Dataview plugin required) and also understands `"folder"`, `[[link]]`, and `AND`/`OR`/`NOT` combinations. The tree has a copy-to-clipboard button if you want the result as a static markdown index.
